src/controller/CurrentFabricRemover.cpp

```diff
@@ -145,7 +145,10 @@ void CurrentFabricRemover::OnCommandFailure(void * context, CHIP_ERROR err)
 
 void CurrentFabricRemover::FinishRemoveCurrentFabric(void * context, CHIP_ERROR err)
 {
-    ChipLogError(Controller, "Remove Current Fabric Result : %" CHIP_ERROR_FORMAT, err.Format());
+    if (err != CHIP_NO_ERROR)
```

Comment:
@agners In the success case we should have left a progress-level log saying it completed successfully...  Please put that back.

Reply:
Ok, #33914 adds a progress-log level on success.

Reply:
Thank you!